packages/@dcl/react-ecs/src/system.ts

```diff
@@ -15,7 +15,7 @@ export function createReactBasedUiSystem(
   pointerSystem: PointerEventsSystem
 ): ReactBasedUiSystem {
   const renderer = createReconciler(engine, pointerSystem)
-  let uiComponent: UiComponent = () => ({})
+  let uiComponent: UiComponent
 
   function ReactBasedUiSystem() {
     renderer.update(uiComponent())
```

Comment:
oops? `cannot call function uiComponent is undefined`

Reply:
what about https://github.com/decentraland/js-sdk-toolchain/pull/336/commits/9f1223c08f02b9e881125d3fb1f460844db33199